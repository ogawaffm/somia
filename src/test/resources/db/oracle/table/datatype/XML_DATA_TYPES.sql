BEGIN
    EXECUTE IMMEDIATE 'DROP TABLE TEST.XML_DATA_TYPES';
EXCEPTION
    WHEN OTHERS THEN
        IF SQLCODE != -942 THEN
            RAISE;
        END IF;
END;
/
CREATE TABLE TEST.XML_DATA_TYPES
(
    XMLTYPE_COL     XMLTYPE,
    URITYPE_COL     URITYPE,
    DBURITYPE_COL   DBURITYPE,
    XDBURITYPE_COL  XDBURITYPE,
    HTTPURITYPE_COL HTTPURITYPE

)