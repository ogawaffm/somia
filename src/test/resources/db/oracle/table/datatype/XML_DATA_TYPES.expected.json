{
  "name": "XML_DATA_TYPES",
  "catalogName": "",
  "schemaName": "TEST",
  "typeName": "TABLE",
  "comment": "",
  "columns": [
    {
      "name": "XMLTYPE_COL",
      "type": {
        "name": "XMLTYPE",
        "typeNumber": 2009,
        "className": "java.sql.SQLXML",
        "precision": 2000,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 10,
        "isCaseSensitive": false,
        "isSearchable": false,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 2147483647,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    },
    {
      "name": "URITYPE_COL",
      "type": {
        "name": "URITYPE",
        "typeNumber": 1111,
        "className": "java.lang.Object",
        "precision": 256,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 10,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 0,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    },
    {
      "name": "DBURITYPE_COL",
      "type": {
        "name": "DBURITYPE",
        "typeNumber": 1111,
        "className": "java.lang.Object",
        "precision": 1,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 10,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 0,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    },
    {
      "name": "XDBURITYPE_COL",
      "type": {
        "name": "XDBURITYPE",
        "typeNumber": 1111,
        "className": "java.lang.Object",
        "precision": 1,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 10,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 0,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    },
    {
      "name": "HTTPURITYPE_COL",
      "type": {
        "name": "HTTPURITYPE",
        "typeNumber": 1111,
        "className": "java.lang.Object",
        "precision": 1,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 10,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 0,
        "baseType": null
      },
      "comment": "",
      "collation": "",
      "defaultDefinition": ""
    }
  ],
  "indexes": [
    {
      "name": "",
      "columns": [],
      "isUnique": true,
      "comment": "",
      "qualifierName": "",
      "isClustered": false,
      "isHashed": false,
      "isStatistic": true,
      "whereClause": "",
      "cardinality": 0,
      "pages": 0
    }
  ],
  "collation": "",
  "primaryKey": null,
  "bestRowIdentifier": null
}